extensions/amp-render/1.0/test/test-amp-render.js

```diff
@@ -358,5 +358,48 @@ describes.realWin(
       expect(options.expr).to.equal('fullName');
       expect(options.refresh).to.be.false;
     });
+
+    it('should perform url replacement in src', async () => {
```

Comment:
Let's also add a test case for `[src]` and per-use opt-in

Reply:
This test case involves clicking a button to mutate src which is not trivial with Karma. Looked at similar tests in `amp-selector` for use of `click()`. But since this involves use of `setState` I haven't yet been able to get it working in these tests.

Reply:
Can we call `mutationObserverCallback` directly? Or am I missing something?